common/thorhelper/thorcommon.hpp

```diff
@@ -278,10 +278,10 @@ class ActivityTimer
     ActivityTimer(ActivityTimeAccumulator &_accumulator, const bool _enabled)
     : accumulator(_accumulator), enabled(_enabled), isFirstRow(false)
     {
-        if (enabled)
+        if (likely(enabled))
         {
             startCycles = get_cycles_now();
-            if (!accumulator.firstRow)
+            if (unlikely(!accumulator.firstRow))
```

Comment:
I dont know if this is more likely or unlikely ?  At first thought I was thinking likely.